main.rb

```diff
@@ -1,11 +1,13 @@
 require_relative './app'
 require_relative './menu'
 require_relative './game_file'
+require_relative './book_data'
 
 def main
   app = App.new
   Dir.exist?('./storage') ? app.read_files : nil
   Dir.exist?('./storage') ? app.read_files2 : nil
+  Dir.exist?('./storage') ? app.read_booklabel : nil
```

Comment:
- :clap: Awesome!!!
- [ ] [OPTIONAL] Since we are working on a project with similar instances, I think we can save this line to avoid the repetition of processes by adding `read_booklabel` to the `switch/case` condition in `read_files2` where we have `author` and `genre` cases. This might make the code look even cleaner and more maintainable. I have attached an image hoping it will help :point_down: 

![image](https://user-images.githubusercontent.com/99715640/214782521-8ad6f636-070a-4aa1-afd8-8c1ebfceb481.png)

Reply:
I tried adding the two cases but it threw up a linter error for me, but let me try to add the book and label cases into `read_files` and `read_files2` respectively